### ISSUE TITLE
Support for short positions

### ISSUE THREAD

**Issue by ForstMaple** [wsyxbcl/Reimu#22]
Short-selling can happen for hedging or speculation purposes. Short positions are unsupported at present.
Given that short-selling is usually not popular or available to the common investors, this will be a low priority and treated as an experimental function. 

Since short-selling is more likely to be used in a portfolio, this issue will depend on #20.
